kernel/src/aarch64/arch_api/init.rs

```diff
@@ -87,4 +90,9 @@ pub fn arch_init() {
             },
         });
     }
+
+    // Manually trigger an exception to test the exception handling.
+    unsafe {
+        core::arch::asm!("svc #0");
+    }
```

Comment:
Sure you want to leave in the exception?

Reply:
I'll remove them both when I've made sure everything is working properly. I just didn't want to take them out before finishing the changes.